tests/AulaTests.hs

```diff
@@ -30,10 +30,11 @@ import Frontend.Prelude as X hiding (get, put)
 
 
 testConfig :: IO Config
-testConfig = (devel & generateDemoData .~ False &) . (listenerPort .~) <$> pop
-  where
-    pop :: IO Int
-    pop = modifyMVar testConfigPortSource $ \(h:t) -> pure (t, h)
+testConfig = do
+    cfg <- getConfig DontWarnMissing
+    pop <- modifyMVar testConfigPortSource $ \(h:t) -> pure (t, h)
+    return (cfg & generateDemoData .~ False & listenerPort .~ pop)
```

Comment:
I think these `&` tend to look much better when aligned:

```
pure $ cfg & generateDemoData .~ False
                 & listenerPort .~ pop
```

This might also work if you like it:

```
cfg & generateDemoData .~ False
    & listenerPort .~ pop
    & pure
```